kafka/src/main/java/org/apache/apex/malhar/kafka/AbstractKafkaInputOperator.java

```diff
@@ -366,17 +374,19 @@ public String getClusters()
     return Joiner.on(';').join(clusters);
   }
 
-  public void setTopics(String... topics)
+  public void setTopics(String topics)
   {
-    this.topics = topics;
+    Topic.validate(topics.replace(" |,", "");
```

Comment:
no, valid topic + valid topic != valid topic. It may exceed length limit of a valid topic for example.